feature-libs/product-configurator/rulebased/components/group-title/configurator-group-title.component.html

```diff
@@ -1,5 +1,10 @@
 <ng-container *ngIf="displayedGroup$ | async as group; else ghostGroup">
   <div class="cx-group-title">
+    <ng-container *ngIf="isMobile() | async as mobile">
```

Comment:
Better
```suggestion
    <ng-container *ngIf="isMobile() | async">
```
Then next container is not needed

Reply:
Agree! Done!